CMake/resolve_dependency_modules/duckdb.cmake

```diff
@@ -24,12 +24,16 @@ resolve_dependency_url(DUCKDB)
 
 message(STATUS "Building DuckDB from source")
 # We need remove-ccache.patch to remove adding ccache to the build command
-# twice. Velox already does this.
+# twice. Velox already does this. We need fix-duckdbversion.patch as DuckDB
+# tries to infer the version via a git commit hash or git tag. This inference
+# can lead to errors when building in another git project such as Prestissimo.
 FetchContent_Declare(
   duckdb
   URL ${VELOX_DUCKDB_SOURCE_URL}
   URL_HASH ${VELOX_DUCKDB_BUILD_SHA256_CHECKSUM}
-  PATCH_COMMAND git apply ${CMAKE_CURRENT_LIST_DIR}/duckdb/remove-ccache.patch)
+  PATCH_COMMAND
+    git apply ${CMAKE_CURRENT_LIST_DIR}/duckdb/remove-ccache.patch && git apply
+    ${CMAKE_CURRENT_LIST_DIR}/duckdb/fix-duckdbversion.patch)
```

Comment:
You could test it with both changes in a single patch, maybe for some reason the git version in the ci buildwheel isn't able to apply them one after another as they both modify the same file?

Reply:
good point. Let me try this.